sympy/geometry/line.py

```diff
@@ -1762,7 +1793,6 @@ class Line2D(LinearEntity2D, Line):
     Examples
     ========
 
-    >>> import sympy
     >>> from sympy import Point
     >>> from sympy.abc import L
     >>> from sympy.geometry import Line, Segment
```

Comment:
You can combine the imports as `from sympy.geometry import Point, Line, Segment`

Reply:
Yes...this is rampant throughout the file but it wasn't introduced by this PR. I'm ok with this as is.